ColorPairUtility.cpp

```diff
@@ -0,0 +1,46 @@
+//
+//  ColorPairUtility.cpp
+//  wellnamed
+//
+//  Created by Bhumika Vaish on 01/05/25.
+//
+
+#include "ColorPairUtility.hpp"
+#include <fstream>
+#include <iostream>
+
+using ColorPairInfo::numberOfMajorColors;
+using ColorPairInfo::numberOfMinorColors;
+
+namespace TelCoColorCoder
+{
+    ColorPair GetColorFromPairNumber(int pairNumber) {
+        int zeroBasedPairNumber = pairNumber - 1;
+        MajorColor majorColor = (MajorColor)(zeroBasedPairNumber / numberOfMinorColors);
+        MinorColor minorColor = (MinorColor)(zeroBasedPairNumber % numberOfMinorColors);
+        return ColorPair(majorColor, minorColor);
+    }
+
+    int GetPairNumberFromColor(MajorColor major, MinorColor minor) {
+        return major * numberOfMinorColors + minor + 1;
+    }
+
+    void PrintColorCodingManual(const std::string& filename) {
```

Comment:
This file/class keeps `PrintColorCodingManual` and the above `Get...` functions together. Consider this: the `Get...` functions rarely change, while the `Print...` function can change frequently, based on a customer's preference (some may want console, others HTML, others pdf...). In general, it is good to separate functionality that has different lifecycles.